code/math/vecmat.cpp

```diff
@@ -1397,7 +1397,7 @@ void compute_point_on_plane(vec3d *q, const plane *planep, const vec3d *p)
 
 	vm_vec_scale_add(q, p, &normal, -k);
 
-	tv = planep->A * q->xyz.x + planep->B * q->xyz.y + planep->C * q->xyz.z + planep->D;
+	// tv = planep->A * q->xyz.x + planep->B * q->xyz.y + planep->C * q->xyz.z + planep->D;
```

Comment:
If I did the math correctly then `tv` is always a constant of `2 * planep->D` (assuming that the position of `q` was  calculated correctly) and as such it isn't very useful.